_pkgdown.yml

```diff
@@ -66,6 +66,10 @@ reference:
       - orderly_location_list
       - orderly_location_remove
       - orderly_location_rename
+  - title: Exporting packets
+    contents:
+      - orderly_export_zip
+      - orderly_import_zip
```

Comment:
Torn here on the names; I wonder about `orderly_zip_import` / `orderly_zip_export` as most of the names work like this (to help with autocomplete). OTOH, if we had different import/export options in future, your way is better.

Reply:
Yeah now that you mention it I think I like those names better. I renamed them, and also renamed the file to `zip.R`.

I think I would have preferred a more generic name instead of `zip`, and then in the future accept a `format = "zip"` argument that could take other values. Sadly the most obvious candidate, `archive`, is already used to mean something completely different. `tarball` kind of works, but it implies the `tar` format which is not much better than `zip` is.